src/Agent.Service/Linux/AgentService.ts

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env node
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+/// <reference path="./definitions/node.d.ts"/>
```

Comment:
I would not compile this.  There's not enough code here (89 lines) to justify pulling in typescript, gulp dev dependencies etc...